src/components/Carousel/Carousel.tsx

```diff
@@ -0,0 +1,32 @@
+import Carousel from "react-multi-carousel";
+
+import "react-multi-carousel/lib/styles.css";
+
+const responsive = {
+	superLargeDesktop: {
+		// the naming can be any, depends on you.
+		breakpoint: { max: 4000, min: 3000 },
+		items: 5,
+	},
+	desktop: {
+		breakpoint: { max: 3000, min: 1024 },
+		items: 3,
+	},
+	tablet: {
+		breakpoint: { max: 1024, min: 464 },
+		items: 2,
+	},
+	mobile: {
+		breakpoint: { max: 464, min: 0 },
```

Comment:
464, a co to za wartość, skąd taką znasz?

Ogólnie do całego pliku, zwykle stosuje się:
480, 768, 1080, 1440, 1920+